frontend/packages/knative-plugin/src/components/overview/KnativeRevisionResources.tsx

```diff
@@ -1,25 +1,32 @@
 import * as React from 'react';
-import { K8sResourceKind } from '@console/internal/module/k8s';
+import { K8sResourceKind, PodKind } from '@console/internal/module/k8s';
+import { PodControllerOverviewItem } from '@console/shared';
 import ConfigurationsOverviewList from './ConfigurationsOverviewList';
 import KSRoutesOverviewList from './RoutesOverviewList';
+import PodsOverview from './PodsOverview';
+import DeploymentOverviewList from './DeploymentOverviewList';
 
 export type KnativeRevisionResourceProps = {
   ksroutes: K8sResourceKind[];
   configurations: K8sResourceKind[];
   obj: K8sResourceKind;
+  pods?: PodKind[];
+  current?: PodControllerOverviewItem;
```

Comment:
Can we name it ```item``` instead of ```current```?

Reply:
i thought of using same prop name as we have for OverviewItems i.e we have current there

Reply:
Then it's ok. Thanks.